### PR TITLE
Upgrade notes to lilac 

### PR DESCRIPTION
**Pull request by NeOneSoft**
This PR upgrades notes plugin to lilac release

### CODE REVIEW

File: tutornotes/templates/notes/build/notes/Dockerfile [hunk 1]
```diff
@@ -5,9 +5,8 @@ RUN apt update && \
   apt upgrade -y && \
   # python 3.8
   apt install -y language-pack-en git python3 python3-pip libmysqlclient-dev
-RUN ln -s /usr/bin/python3 /usr/bin/python \
-    && ln -s /usr/bin/pip3 /usr/bin/pip
 
+RUN ln -s /usr/bin/python3 /usr/bin/python
```
Comment (regisb) [inline tutornotes/templates/notes/build/notes/Dockerfile:9]: I wonder why the `pip` symlink is no longer necessary... Maybe we can also get rid of the `python` symlink?

**Review by regisb — APPROVED** [review]
Looking good! I assume you checked everything is working as expected?

File: tutornotes/templates/notes/build/notes/Dockerfile (hunk 1 above)
Comment (NeOneSoft) [reply tutornotes/templates/notes/build/notes/Dockerfile:9]: I tried @regisb  but I got some errors

**Comment by NeOneSoft** [comment]
Yes Regis!! everything is working as expected!!

**Comment by regisb** [comment]
Thanks for the PR @NeOneSoft! This is now merged in the lilac branch. We'll merge this once the actual Lilac release is out.

**Comment by NeOneSoft** [comment]
Thank you @regisb I'll keep on working!!= )

File: tutornotes/templates/notes/build/notes/Dockerfile (hunk 1 above)
Comment (regisb) [reply tutornotes/templates/notes/build/notes/Dockerfile:9]: Yes, I observed the same thing as you I think: https://github.com/overhangio/tutor-notes/pull/4

**Pull request merged**